packages/commerce-sdk-react/src/hooks/ShopperBaskets/mutation.ts

```diff
@@ -108,95 +105,55 @@ If the current shopper doesn't have an active basket, and the createDestinationB
-   * @see {@link https://tanstack.com/query/latest/docs/react/reference/useMutation | TanStack Query `useMutation` reference} for more information about the return value.
    */
     MergeBasket: 'mergeBasket',
     /**
      * Removes a basket.
-     * @returns A TanStack Query mutation hook for interacting with the Shopper Baskets `deleteBasket` endpoint.
-     * @see {@link https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-baskets?meta=deleteBasket| Salesforce Developer Center} for more information about the API endpoint.
-     * @see {@link https://salesforcecommercecloud.github.io/commerce-sdk-isomorphic/classes/shopperbaskets.shopperbaskets-1.html#deletebasket | `commerce-sdk-isomorphic` documentation} for more information on the parameters and returned data type.
-     * @see {@link https://tanstack.com/query/latest/docs/react/reference/useMutation | TanStack Query `useMutation` reference} for more information about the return value.
      */
     DeleteBasket: 'deleteBasket',
     /**
    * Updates a basket. Only the currency of the basket, source code, the custom
 properties of the basket, and the shipping items will be considered.
-   * @returns A TanStack Query mutation hook for interacting with the Shopper Baskets `updateBasket` endpoint.
-   * @see {@link https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-baskets?meta=updateBasket| Salesforce Developer Center} for more information about the API endpoint.
-   * @see {@link https://salesforcecommercecloud.github.io/commerce-sdk-isomorphic/classes/shopperbaskets.shopperbaskets-1.html#updatebasket | `commerce-sdk-isomorphic` documentation} for more information on the parameters and returned data type.
-   * @see {@link https://tanstack.com/query/latest/docs/react/reference/useMutation | TanStack Query `useMutation` reference} for more information about the return value.
    */
     UpdateBasket: 'updateBasket',
     /**
      * Sets the billing address of a basket.
-     * @returns A TanStack Query mutation hook for interacting with the Shopper Baskets `updateBillingAddressForBasket` endpoint.
-     * @see {@link https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-baskets?meta=updateBillingAddressForBasket| Salesforce Developer Center} for more information about the API endpoint.
-     * @see {@link https://salesforcecommercecloud.github.io/commerce-sdk-isomorphic/classes/shopperbaskets.shopperbaskets-1.html#updatebillingaddressforbasket | `commerce-sdk-isomorphic` documentation} for more information on the parameters and returned data type.
-     * @see {@link https://tanstack.com/query/latest/docs/react/reference/useMutation | TanStack Query `useMutation` reference} for more information about the return value.
```

Comment:
So the deletion of these type doc callouts is repeated many times throughout the PR. And I see we added a plugin to kinda "bypass the linter" so to speak. What's the motivation here? Is it the correct direction to drop these kinds of links which might point to useful docs about the response shape / schema, etc.?

Reply:
The links show up on the enum page, for example, https://salesforcecommercecloud.github.io/pwa-kit/enums/ShopperBasketsMutations.html

The same links are repeated 20+ times on the same page, which probably isn't what we want.